src/egeoip.app.src

```diff
@@ -3,6 +3,12 @@
   {vsn,"1.1"},
   {modules,[]},
   {registered, []},
-  {env, [{dbfile, city}]},
+  {env, [
+      {dbfile, city}%,
+      % dbpath may be left undefined, so as to not run the self-updater
```

Comment:
This PR only handles remote files (in format .gz), which is our use case and current concern. If we have a use case for local files, sure, we can do another PR and support that.